docs/src/howto-guides/010_add_self_signed_trusted.md

```diff
@@ -0,0 +1,27 @@
+# How to add self-signed certificate to trusted
+
+To manually add a self-signed certificate to be trusted by Linux system:
+
+Create a `/usr/local/share/ca-certificates/` directory if it does not exist on your computer:
+
+```shell
+mkdir /usr/local/share/ca-certificates/
+```
+
+Copy your root certificate (.crt file) to the created directory:
+
+```shell
+cp <full_path_to_the_certificate> /usr/local/share/ca-certificates/
+```
+
+Update the certificates:
+
+```shell
+sudo update-ca-certificates
```

Comment:
When you run this command, it puts the certificate into some other file / directory to be "trusted".

It would be good to add the command to check that the certificate has been successfully added to that file / directory.

Reply:
`update-ca-certificates` will add the result in the output, I updated the command with sample output.

Reply:
OK. But I've seen someone run this command and it said like "233 added". How can a user check their new certificate is installed correctly?